docs_user/assemblies/assembly_planning-your-ipam-configuration.adoc

```diff
@@ -6,65 +6,39 @@ ifdef::context[:parent-context: {context}]
-number of IP addresses available for services, it may even be impossible to
-reuse the existing IP ranges used in adopted environment, depending on its
-size. Prudent planning is required to determine which options are available in
-your particular case.
-
-The total number of IP addresses required for the new control plane services,
-in each isolated network, is calculated as a sum of the following:
-
-* The number of {OpenShiftShort} worker nodes. (Each node will require 1 IP address in
-  `NodeNetworkConfigurationPolicy` custom resources (CRs).)
-* The number of IP addresses required for the data plane nodes. (Each node will require
-  an IP address from `NetConfig` CRs.)
-* The number of IP addresses required for control plane services. (Each service
-  will require an IP address from `NetworkAttachmentDefinition` CRs.) This
-  number depends on the number of replicas for each service.
-* The number of IP addresses required for load balancer IP addresses. (Each
-  service will require a VIP address from `IPAddressPool` CRs.)
-
-As of the time of writing, the simplest single worker node {OpenShiftShort} deployment
-(CRC) has the following IP ranges defined (for the `internalapi` network):
-
-* 1 IP address for the single worker node;
-* 1 IP address for the data plane node;
+number of IP addresses available for services, you cannot reuse the existing IP ranges in the {rhos_acro} environment, depending on the size.
+
+The total number of IP addresses that are required for the new control plane services in each isolated network is calculated as the sum of the following:
+
+* The number of {OpenShiftShort} worker nodes. Each node requires 1 IP address in the `NodeNetworkConfigurationPolicy` custom resources (CRs).
+* The number of IP addresses required for the data plane nodes. Each node requires an IP address from the `NetConfig` CRs.
+* The number of IP addresses required for control plane services. Each service requires an IP address from the `NetworkAttachmentDefinition` CRs. This number depends on the number of replicas for each service.
+* The number of IP addresses required for load balancer IP addresses. Each service requires a Virtual IP address from the `IPAddressPool` CRs.
+
+For example, a simple single worker node {OpenShiftShort} deployment
+with Red Hat OpenShift Local has the following IP ranges defined for the `internalapi` network:
+
+* 1 IP address for the single worker node
+* 1 IP address for the data plane node
 * `NetworkAttachmentDefinition` CRs for control plane services:
-  `X.X.X.30-X.X.X.70` (41 addresses);
+  `X.X.X.30-X.X.X.70` (41 addresses)
 * `IPAllocationPool` CRs for load balancer IPs: `X.X.X.80-X.X.X.90` (11
-  addresses).
+  addresses)
 
-Which comes to a total of 54 IP addresses allocated to the `internalapi`
+This example shows a total of 54 IP addresses allocated to the `internalapi`
 allocation pools.
 
 // TODO: update the numbers above for a more realistic multinode cluster.
```

Comment:
Has the TODO comment been addressed?

Reply:
@booxter It's okay to leave the TODO comments. I've just been calling them out to remind engineers to either address them, or if they have been addressed, to delete them.